client/blocks/login/index.jsx

```diff
@@ -58,15 +59,20 @@ class Login extends Component {
 			rememberMe,
 		} = this.state;
 
-		if ( twoStepNonce && twoFactorAuthType === 'code' ) {
+		if ( twoStepNonce && ( twoFactorAuthType === 'code' || twoFactorAuthType === 'sms' ) ) {
 			return (
-				<VerificationCodeForm rememberMe={ rememberMe } onSuccess={ this.rebootAfterLogin } />
+				<VerificationCodeForm
+					rememberMe={ rememberMe }
+					onSuccess={ this.rebootAfterLogin }
+					twoFactorAuthType={ twoFactorAuthType } />
 			);
 		}
 
 		if ( twoStepNonce && twoFactorAuthType === 'push' ) {
 			return (
-				<WaitingTwoFactorNotificationApproval onSuccess={ this.rebootAfterLogin } />
+				<WaitingTwoFactorNotificationApproval
+					onSuccess={ this.rebootAfterLogin }
+					twoFactorAuthType={ twoFactorAuthType } />
```

Comment:
`WaitingTwoFactorNotificationApproval` doesn't need a `twoFactorAuthType` prop because the value will always be `push`.